code/modules/antagonists/nukeop/datums/operative_support.dm

```diff
@@ -0,0 +1,55 @@
+/datum/antagonist/nukeop/support
+	name = ROLE_OPERATIVE_OVERWATCH
+	show_to_ghosts = TRUE
+	send_to_spawnpoint = TRUE
+	nukeop_outfit = /datum/outfit/syndicate/support
+
+/datum/antagonist/nukeop/support/greet()
+	owner.current.playsound_local(get_turf(owner.current), 'sound/machines/printer.ogg', 100, 0, use_reverb = FALSE)
+	to_chat(owner, span_big("You are a [name]! You've been temporarily assigned to provide camera overwatch and manage communications for a nuclear operative team!"))
+	to_chat(owner, span_red("Use your tools to set up your equipment however you like, but do NOT attempt to leave your outpost."))
+	owner.announce_objectives()
+
+/datum/antagonist/nukeop/support/on_gain()
+	..()
+	for(var/datum/mind/teammate_mind in nuke_team.members)
+		var/mob/living/our_teammate = teammate_mind.current
+		our_teammate.AddComponent( \
```

Comment:
We're not checking if they've already got one before giving them another, yeah it's a component but i dunno.

Reply:
that component is `COMPONENT_DUPE_HIGHLANDER` anyways (though i guess it should be `COMPONENT_DUPE_SELECTIVE` to only allow dupes on different networks, i didn't think of that)